test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
- [ ] This file is supposed to be in `.gitignore` which means when you push to Github it won't be pushed with the rest of the files. Kindly fix this issue. Hint: check this [thread](https://stackoverflow.com/questions/1274057/how-do-i-make-git-forget-about-a-file-that-was-tracked-but-is-now-in-gitignore) for more information.